Add 3.6.1 blog post (#1084)
godotengine/godot-website: collections/_article/maintenance-release-godot-3-6-1.md

---
title: "Maintenance release: Godot 3.6.1"
excerpt: "Godot 3.6 was released in September last year, and it has been mostly stable, although there have been a few more important bugs which warrant a maintenance \"patch\" release (3.6.x)."
categories: ["release"]
author: lawnjelly
image: /storage/app/uploads/public/631/1d7/7cc/6311d77cc7f5f571027656.jpg
date: 2025-06-24 12:00:00
---

[Godot 3.6](/article/godot-3-6-finally-released) was released in September last year, and it has been mostly stable, although there have been a few more important bugs which warrant a maintenance "patch" release (3.6.x).

> As well as bug fixes this release also contains some cherry-picks from 4.x to update libraries, so let us know if you spot any regressions.

Reviewer: ```suggestion
As well as bug fixes, this release also contains some cherry-picks from 4.x to update libraries. Let us know if you spot any regressions.
```

[Jump to the **Downloads** section.](#downloads)

As usual, you can try it live with the [**online version of the Godot editor**](https://editor.godotengine.org/releases/3.6.1/) updated for this release.

> ## Major Bugs Fixed

Reviewer 2: ```suggestion
## Major bugs fixed
```

> ### Grid Snapping + Ruler in Editor

Reviewer 2: ```suggestion
### Grid snapping + ruler in editor
```

A small change to `Range::set_value` unfortunately led to a major regression in grid and ruler behaviour when zoomed in ([GH-98466](https://github.com/godotengine/godot/issues/98466)).

We decided on balance the safest course of action in 3.x was to revert the change.

- GUI: Revert changes to `Range::set_value` ([GH-100459](https://github.com/godotengine/godot/pull/100459)).

### Performance regression due to directional shadow fade_start

> Users noticed a drop in performance  in 3D scenes with `DirectionalLight` in Godot 3.6 compared to 3.5. We tracked this down to the introduction of the `fade_start` property, which made some changes to the shader.

Reviewer: ```suggestion
Users noticed a drop in performance in 3D scenes with DirectionalLight in Godot 3.6 compared to 3.5. We tracked this down to the introduction of the `fade_start` property, which made some changes to the shader.
```

## Bug reports

As a tester, you are encouraged to [open bug reports](https://github.com/godotengine/godot/issues) if you experience issues with 3.6.1. Please check first the [existing issues on GitHub](https://github.com/godotengine/godot/issues), using the search function with relevant keywords, to ensure that the bug you experience is not known already.

In particular, any change that would cause a regression in your projects is very important to report (e.g. if something that worked fine in 3.6 or earlier 3.5.x releases no longer works in 3.6.1).

## Support

Godot is a non-profit, open source game engine developed by hundreds of contributors on their free time, and a handful of part or full-time developers, hired thanks to [donations from the Godot community](/donate). A big thankyou to everyone who has contributed [their time](https://github.com/godotengine/godot/blob/master/AUTHORS.md) or [financial support](https://github.com/godotengine/godot/blob/master/DONORS.md) to the project!

> If you'd like to support the project financially and help us secure our future hires, you can do so on [Patreon](https://www.patreon.com/godotengine) or [PayPal](/donate).

Reviewer: Use the fund link:

```suggestion
If you'd like to support the project financially and help us secure our future hires, you can do so on the [Development Fund](https://fund.godotengine.org).
```

---
title: "Maintenance release: Godot 3.6.1"
excerpt: "Godot 3.6 was released in September last year, and it has been mostly stable, although there have been a few more important bugs which warrant a maintenance \"patch\" release (3.6.x)."
categories: ["release"]
author: lawnjelly
image: /storage/blog/covers/maintenance-release-godot-3-6-1.webp
image_caption_title: CraftCraft
image_caption_description: A game by Placeholder Gameworks
date: 2025-06-25 12:00:00
---

[Godot 3.6](/article/godot-3-6-finally-released) was released in September last year, and it has been mostly stable, although there have been a few more important bugs which warrant a maintenance "patch" release (3.6.x).

As well as bug fixes, this release also contains some cherry-picks from 4.x to update libraries. Let us know if you spot any regressions.

[Jump to the **Downloads** section.](#downloads)

> As usual, you can try it live with the [**online version of the Godot editor**](https://editor.godotengine.org/releases/3.6.1/) updated for this release.

Author: This link is also not working currently, maybe we haven't built online version yet (and it might need to be 3.6.1-stable rather than 3.6.1).

- mbedtls: Update to upstream version 2.28.9 ([GH-100013](https://github.com/godotengine/godot/pull/100013)).
- mbedTLS: Update to version 2.28.10 ([GH-104580](https://github.com/godotengine/godot/pull/104580)).
- certs: Sync with upstream as of Apr 8 2025 ([GH-106615](https://github.com/godotengine/godot/pull/106615)).
- Fix unzSeekCurrentFile not resetting total_out_64 ([GH-106872](https://github.com/godotengine/godot/pull/106872)).
- libwebm: Fix double free in mkvparser ContentEncoding ([GH-107781](https://github.com/godotengine/godot/pull/107781)).

## Changelog

**21 contributors** submitted around **45 fixes** for this release. See our [**interactive changelog**](https://godotengine.github.io/godot-interactive-changelog/#3.6.1) for the complete list of changes since the 3.6 release.

This release is built from commit [`b1ba98fce`](https://github.com/godotengine/godot/commit/b1ba98fced19ac05b7a39b64a97dd7b1005cb7bb).

<a id="downloads"></a>
## Downloads

The downloads for this dev snapshot can be found directly on our repository:

- [Standard build](https://github.com/godotengine/godot-builds/releases/3.6.1) (GDScript, GDNative, VisualScript).

> - [Mono build](https://github.com/godotengine/godot-builds/releases/3.6.1) (C# support + all the above). You need to have dotnet CLI or MSBuild installed to use the Mono build. Relevant parts of Mono **6.12.0.182** are included in this build.

Reviewer 3: Bad links, "page not found" for me. 🙃

Author: I opened this PR to correct the links:
https://github.com/godotengine/godot-website/pull/1106

It's kind of chicken and egg, I made the draft before I knew what the links would be.
